Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -963,31 +967,28 @@ - (CGSize)quotedMessageSizeForContentWidth:(int)contentWidth includeMargins:(BOO
         [OWSQuotedMessageView quotedMessageViewForConversation:self.viewItem.quotedReply
                                          displayableQuotedText:displayableQuotedText
                                                     isOutgoing:isOutgoing];
-    const int maxMessageWidth = [self maxMessageWidthForContentWidth:contentWidth];
-    CGSize result = [quotedMessageView sizeForMaxWidth:maxMessageWidth - kBubbleThornSideInset];
-    if (includeMargins) {
-        result.width += kBubbleThornSideInset;
-    }
-
-    return result;
+    CGSize result = [quotedMessageView sizeForMaxWidth:self.layoutInfo.maxMessageWidth];
+    return CGSizeCeil(result);
 }
 
-- (CGSize)sizeForContentWidth:(int)contentWidth
+- (CGSize)measureSize
 {
+    OWSAssert(self.layoutInfo);
+    OWSAssert(self.layoutInfo.viewWidth > 0);
     OWSAssert(self.viewItem);
     OWSAssert([self.viewItem.interaction isKindOfClass:[TSMessage class]]);
 
     CGSize cellSize = CGSizeZero;
 
-    CGSize quotedMessageSize = [self quotedMessageSizeForContentWidth:contentWidth includeMargins:YES];
+    CGSize quotedMessageSize = [self quotedMessageSize];
     cellSize.width = MAX(cellSize.width, quotedMessageSize.width);
     cellSize.height += quotedMessageSize.height;
 
-    CGSize mediaContentSize = [self bodyMediaSizeForContentWidth:contentWidth];
+    CGSize mediaContentSize = [self bodyMediaSize];
     cellSize.width = MAX(cellSize.width, mediaContentSize.width);
     cellSize.height += mediaContentSize.height;
 
-    CGSize textContentSize = [self bodyTextSizeForContentWidth:contentWidth includeMargins:YES];
+    CGSize textContentSize = [self bodyTextSize:YES];
```

Comment:
nit: Can you label the parameter in the message signature e.g. `bodyTextSizeWithIncludeMargins:YES`?

Reply:
Done.